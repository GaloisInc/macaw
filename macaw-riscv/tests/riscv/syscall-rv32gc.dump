
syscall-rv32gc.exe:     file format elf32-littleriscv


Disassembly of section .text:

00010074 <_start>:
   10074:	1141                	c.addi	sp,-16
   10076:	c606                	c.swsp	ra,12(sp)
   10078:	c422                	c.swsp	s0,8(sp)
   1007a:	0800                	c.addi4spn	s0,sp,16
   1007c:	2031                	c.jal	10088 <__getpid>
   1007e:	0001                	c.addi	zero,0
   10080:	40b2                	c.lwsp	ra,12(sp)
   10082:	4422                	c.lwsp	s0,8(sp)
   10084:	0141                	c.addi	sp,16
   10086:	8082                	c.jr	ra

00010088 <__getpid>:
   10088:	0ac00893          	addi	a7,zero,172
   1008c:	00000073          	ecall
   10090:	8082                	c.jr	ra
	...
