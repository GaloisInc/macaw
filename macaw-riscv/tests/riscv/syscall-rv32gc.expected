R { fileEntryPoint = Nothing
  , funcs = [ (0x10074, [(0x10074, 10), (0x1007e, 10)])
            , (0x10088, [(0x10088, 8), (0x10090, 2)]) ]
  , ignoreBlocks = []
  }
